src/find_secrets_in_vault/find_secrets_in_vault.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+"""
+Vault Secrets Search Script
+
+This script searches for a specific term in all Vault secrets.
+
+Usage:
+    python find_secrets_in_vault.py <search_term>
+    
+Environment Variables (required):
+    VAULT_HOST: The Vault hostname
+    VAULT_PORT: The Vault port
+    VAULT_KVV2_MOUNT_POINT: The KVv2 mount point
+    VAULT_PATH: The secret path
+
+Arguments:
+    search_term: the text to search for in secret values
+
+Example:
+    # connect to VPN
+    # download .env.shared for env vars and set them (artsy specific)
+    export VAULT_HOST=foo.example.com
+    export VAULT_PORT=1234
+    export VAULT_KVV2_MOUNT_POINT=foo
+    export VAULT_PATH=/foo/bar
+    python find_secrets_in_vault.py "database_credential"
+    python find_secrets_in_vault.py "api_key"
+
+The script will automatically:
+- Connect to Vault using the specified host and port
+- Login to Vault using AWS authentication
+- Search through all vault secrets for the term
+"""
+
+import subprocess
+import sys
+import os
+
+if len(sys.argv) < 2:
+    print("Usage: find_secrets_in_vault.py <search_term>")
+    print("Make sure to set VAULT_HOST VAULT_PORT VAULT_KVV2_MOUNT_POINT and VAULT_PATH environment variables.")
+    sys.exit(1)
+
+to_find = sys.argv[1]
+
+vault_host = os.environ.get('VAULT_HOST')
+vault_port = os.environ.get('VAULT_PORT')
+vault_kvv2_mount_point = os.environ.get('VAULT_KVV2_MOUNT_POINT')
+vault_path = os.environ.get('VAULT_PATH')
+
+if not vault_host:
+    print("Error: VAULT_HOST environment variable is required.")
+    sys.exit(1)
+
+if not vault_port:
+    print("Error: VAULT_PORT environment variable is required.")
+    sys.exit(1)
+
+if not vault_kvv2_mount_point:
+    print("Error: VAULT_KVV2_MOUNT_POINT environment variable is required.")
+    sys.exit(1)
+
+if not vault_path:
+    print("Error: VAULT_PATH environment variable is required.")
+    sys.exit(1)
+
+vault_addr = f"https://{vault_host}:{vault_port}"
+print(f"Setting VAULT_ADDR to {vault_addr}")
+os.environ['VAULT_ADDR'] = vault_addr
```

Comment:
suggestion: the env var could just be VAULT_ADDR

Reply:
Could you elaborate on this one a bit more 🙏, im not following what you are suggesting.

Reply:
Similarly as with other vars, I ended up following the pattern of other script to split up host and port but its minor I think and removes some lines so done in https://github.com/artsy/opstools/pull/151/commits/672af63339dcdc46ea403a33a9566e49ed10de5e